pkgs/development/ocaml-modules/ssl/default.nix

```diff
@@ -1,30 +1,38 @@
-{ lib, buildDunePackage, fetchFromGitHub, pkg-config, openssl
-, dune-configurator }:
+{ alcotest
+, buildDunePackage
+, dune-configurator
+, fetchFromGitHub
+, lib
+, openssl
+, pkg-config
+}:
 
 buildDunePackage rec {
   pname = "ssl";
-  version = "0.5.10";
+  version = "0.5.11";
 
   src = fetchFromGitHub {
     owner = "savonet";
     repo = "ocaml-ssl";
-    rev = "v${version}";
-    sha256 = "1rszqiqayh67xlwd5411k8vib47x9kapdr037z1majd2c14z3kcb";
+    rev = version;
+    sha256 = "sha256-uFr+XSKDGMHaM2o5DODYmt7+LkhnDzzlVX//CtAXBm4=";
   };
 
-  useDune2 = true;
   nativeBuildInputs = [ pkg-config ];
   buildInputs = [ dune-configurator ];
-  propagatedBuildInputs = [openssl];
+  propagatedBuildInputs = [ openssl ];
+
+  doCheck = true;
```

Comment:
```suggestion
  doCheck = lib.versionAtLeast ocaml.version "4.05";
```

Reply:
I want to understand the process. Why `"4.06"`? Where does it come from? How did you figure out that?

Reply:
Don’t forget to take `ocaml` as argument.

Reply:
> Did you build all versions?

Yes.

> Is there a good way to build all versions?

None that I am aware of.

> I mostly use `nixpkgs-review pr` and test latest.

If you had run `nixpkgs-review` this time, you would have spotted that `opa` was gone. Investigating this lead to the suggested fix.